docs/source/notebook-components/weaver_example.ipynb

```diff
@@ -10,25 +10,40 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #28.    if not href.suffix == ".txt":
</pre><p>use <code>!=</code> </p><p> </p><p>The invalid extension should be raised. It is part of the test that the requested output format is respected. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/pavics-sdi/pull/359/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>